bin/MadGraph5_aMCatNLO/cards/production/13TeV/HZaTo2l2g/HZaTo2l2g_customizecards.dat

```diff
@@ -0,0 +1,5 @@
+set param_card mass 9000005 20
+set param_card decay 9000005 auto
+set ptl 0.0  
+set etal -1.0
+set mmll 4.0 
```

Comment:
Please remove the run card settings in customize cards, and implement them to run card directly